app/api/wallet/mnemonic.test.ts

```diff
@@ -12,9 +11,12 @@ describe('getMnemonicHdNodeProvider', () => {
     const data: WalletData = {
       version: "v1",
       type: WalletType.MNEMONIC_UNPROTECTED,
-      raw: "408b285c123836004f4b8842c89324c1f01382450c0d439af345ba7fc49acf705489c6fc77dbd4e3dc1dd8cc6bc9f043db8ada1e243c4a0eafb290d399480840"
+      raw: JSON.stringify([
```

Comment:
Not the most optimized, storing it as json array here for simplicity.
The raw `words` doesn't seems to be needed, I just trim it.